docs/UserGuide.md

```diff
@@ -594,18 +600,60 @@ If you wish, please refer to the below section for a more succinct [command summ
+
 
 --------------------------------------------------------------------------------------------------------------------
 
 ## Glossary
-
-| Term   | Definition                                                                                                                                                                                                                                                                      |
-|--------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| CLI    | Command Line Interface (CLI) is a text-based user interface that allows users to interact with a computer program by typing in commands through a keyboard. <br/> Since a mouse is not needed for CLI, it is often **a more efficient interaction method for fast typists.**    |
+[Back to Table of Contents](#table-of-contents)
+
+| Term      | Definition                                                                                                                                                                                                                                                                   |
+|-----------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| CLI       | Command Line Interface (CLI) is a text-based user interface that allows users to interact with a computer program by typing in commands through a keyboard. <br/> Since a mouse is not needed for CLI, it is often **a more efficient interaction method for fast typists.** |
+| GUI       | Graphical User Interface (GUI) is a user interface that allows users to interact with the application using graphical elements like text fields, buttons, and menus.                                                                                                         |
+| Parameter | Parameters are specific settings for you to customise your command. For example, the name and phone number of a musician are parameters.                                                                                                                                     |
+| Command   | A command is an instruction given to the application to perform a specific action. For example, `list` is a command that displays all musicians and bands.                                                                                                                   |
```

Comment:
I think the glossary part also has merge conflict with my previous PR, but I think yours is better so feel free to replace mine!